pkg/cmd/issue/view/view.go

```diff
@@ -89,7 +92,7 @@ func NewCmdView(f *cmdutil.Factory, runF func(*ViewOptions) error) *cobra.Comman
 
 var defaultFields = []string{
 	"number", "url", "state", "createdAt", "title", "body", "author", "milestone",
-	"assignees", "labels", "projectCards", "reactionGroups", "lastComment", "stateReason",
+	"assignees", "labels", "reactionGroups", "lastComment", "stateReason",
```

Comment:
Do we check the case in which `projectCards` is provided as JSON field? Or maybe we don't need it?

Reply:
See https://github.com/cli/cli/issues/10714

> This does not affect cases in which the user has used the --json flag, or gh api.